doc/sphinx-guides/source/installation/config.rst

```diff
@@ -267,8 +305,8 @@ You also have the option to set a custom expiration length, in seconds, for a ge
 In this example, you would be setting the expiration length for one hour.
 
 
-Setting up Compute
-+++++++++++++++++++
+Setting up Compute with Swift
```

Comment:
I appreciate this clarification that the Compute button is currently tied to Swift. 😄